.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,60 @@
+name: Commitlint
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+  merge_group:
+    branches:
+      - main
+    types:
+      - checks_requested
+jobs:
+  commit-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install conventional-commit linter
+        run: npm install @commitlint/config-conventional @commitlint/cli
```

Comment:
Is it necessary to install `@commitlint/cli` if you use `npx commitlint`?

Reply:
config-conventional is only a config, and commitlint/cli is the actual program.

Reply:
Commitlint's docs say to run commitlint like that. I am not familiar with npx, I was just sticking to their examples, so I cannot reply to you. But I'd like to stick to their practices.
https://commitlint.js.org/#/guides-ci-setup?id=github-actions